lib/Logger/Logger.cpp

```diff
@@ -529,6 +531,12 @@ void Logger::log(char const* logid, char const* function, char const* file, int
       dumper.appendString(_hostname.data(), _hostname.size());
     }
 
+    // meta data from log context
+    logContext.each([&out](std::string const& key, std::string const& value) {
+      out.append(",\"", 2).append(key).append("\":\"", 3).append(value);
```

Comment:
Here we need to ensure that at least the values are properly JSON-encoded, but ideally we also escape all keys here to be ready for unexpected things. See above for the usage of `dumper.appendString()`.